Gemfile

```diff
@@ -1,4 +1,4 @@
-source 'https://rubygems.org'
+source 'http://rubygems.org'
```

Comment:
Using an insecure source is probably not a good idea. Is this due to an HTTPS issue on a developer machine?